src/disklayout.js

```diff
@@ -46,11 +47,26 @@ dolayout(disks, layout, nspares, excluded, enable_cache, width)
 			return (true);
 		});
 	} else if (os.type() === 'Linux') {
+		var zvolExp = /^zd[0-9a-z]*$/;
```

Comment:
If the device is "zd" all by itself, it'll match this expression.  Is that desirable? If not, change the `*` to `+`.

Reply:
Good point, I've pushed that change, thanks!